src/components/barchart/BarChart.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import styled from "styled-components";
+import theme from "../../theme";
+import { space } from "styled-system";
+
+interface IBarValue {
+  color: string;
+  value: number;
+}
+
+interface IBarValues {
+  values: IBarValue[];
+}
+
+interface IStyledBarProps {
+  height?: string;
+}
+
+const StyledBarChart = styled.svg`
```

Comment:
It's better to declare props here -> styled.svg<IStyledBarProps>
right now you would have to specify props type every time you want to use it to apply styles